scheduler/test/cook/test/mesos.clj

```diff
@@ -18,11 +18,8 @@
   (:require [clojure.core.async :as async]
             [cook.curator :as curator]
             [cook.datomic]
-            [cook.mesos :as mesos]
```

Comment:
We should this cleanup in a separate PR.

Reply:
:+1: done